FSJ_django20_project/FSJ/templates/FSJ/coord_student_list.html

```diff
@@ -32,5 +32,9 @@
 {% endfor %}
 </table>
 </div>
+<div>
+{# Translators: Button to add student #}
+ <button class = "addbutton"><a class = "header" href="/studentlist/add/"</a>{% trans "Add Student" %}</button>
```

Comment:
should change to

<button class = "addbutton" onclick="window.location.href='/studentlist/add/'">{% trans "Add student" %}</button> 

to implement onclick. Makes whole button a link, works on Pierre's machine and Dave's